python/private/pypi/whl_library_targets.bzl

```diff
@@ -103,7 +103,14 @@ def whl_library_targets(
     for filegroup_name, glob in filegroups.items():
         native.filegroup(
             name = filegroup_name,
-            srcs = native.glob(glob, allow_empty = True),
+            srcs = native.glob(
+                glob,
+                exclude = [
+                    # File names with spaces should be excluded.
+                    "**/* *",
```

Comment:
I thought that our supported `bazel` versions support files with spaces, so why do we need to exclude them?

Reply:
However, it seems that someone tried it and it did not work?

https://github.com/michael-christen/toolbox/pull/184/files

Reply:
Yeah we also hit the issue with setuptools in runfiles, but with the Go runfiles library. Setuptools seems to contain files with spaces, so even if bazel itself can handle then now, the runfiles libraries can't.